qemu/tests/x86_cpu_model.py

```diff
@@ -72,6 +72,17 @@ def run(test, params, env):
     cmd = params["get_model_cmd"]
     out = session.cmd_output(cmd)
     if not re.search(guest_model, out):
+        check_cmd = params.get("check_cmd")
+        if check_cmd:
+            vulnerabilities = params.get("check_items").split(" ")
+            for item in vulnerabilities:
+                h_out = process.getoutput(check_cmd % item).strip()
+                g_out = session.cmd_output(check_cmd % item).strip()
+                if h_out != g_out:
+                    test.fail("Guest %s are not equal to Host %s"
+                              % (h_out, g_out))
+                if item != '*' and g_out != 'Not affected':
+                    test.fail("%s can't get 'Not affected'" % item)
```

Comment:
So this means all the guest output should be `'Not affected'`? I do not try on guest but on host it seems not like this.

Reply:
> So this means all the guest output should be `'Not affected'`? I do not try on guest but on host it seems not like this.

What's your host's cpu model? we need to test Cooperlake.

Reply:
@nanliu-r Okay I see, my host is not Cooperlake.